hugo/data/categories.yml

```diff
@@ -1,6 +1,37 @@
+#
+# README :
+#
+# It is accepted two different kinds of projects:
+# - without documentation -> the project card will point to the repository source code
+# - having documentation generated by docsrv -> the project card will point to the docs
+#
+# PROJECT WITHOUT DOCS schema:
+#
+#  - name: short name of the project as it will appear in the card title
+#    url: link to the repository source code (example: //github.com/src-d/kmcuda)
+#    desc: project description that will appear in the project card ('/projects' page)
+#    logo: (optional) svg name of the icon (under 'static/img/icons') without the '.svg' extension.
+#          If it's not provided, it will use a random codepill
+#
+# PROJECT HAVING DOCS schema:
```

Comment:
Suggestion: for each field, let the user know the "options" they have or the exact format that is (not) allowed.